### ISSUE TITLE
Deprecated Package

### ISSUE THREAD

**Issue by demarley** [cms-mual/Alignment#26]
``` 
#####################################################################
 # WARNING: the module CondCore.DBCommon.CondDBSetup is deprecated.  #
 # Please import CondCore.CondDB.CondDB_cfi                          #
 #####################################################################
```

**Comment by demarley** [comment]
https://github.com/cms-sw/cmssw/pull/22346/files#diff-1792bfde278f7120bc927ab24625228bL6

**Comment by demarley** [comment]
Fixed issue in only instance that I could find, but the warning still appears -- more digging.

**Comment by demarley** [comment]
As far as I can tell, the warning is no longer in our framework.  It does exist in other places, though:
- [MuonAlignment](https://github.com/cms-sw/cmssw/blob/CMSSW_10_0_2/Alignment/MuonAlignment/python/convertXMLtoSQLite_cfg.py#L18)
- [MuonAlignment](https://github.com/cms-sw/cmssw/blob/CMSSW_10_0_2/Alignment/MuonAlignment/python/convertSQLitetoXML_cfg.py#L10)

**Comment by demarley** [comment]
Fixed in #29 in all parts of our code (as far as I can tell).
